corehq/apps/cloudcare/static/cloudcare/js/formplayer/users/views.js

```diff
@@ -92,13 +92,13 @@ hqDefine("cloudcare/js/formplayer/users/views", function () {
             paginationGoButton: '#pagination-go-button',
             paginationGoText: '#goText',
             paginationGoTextBox: '.module-go-container',
-            perPage: '.js-page-limit',
+            casesPerPageLimit: '.cases-per-page-limit',
```

Comment:
This particular rename seems strange, since the items in this view aren't cases, they're users.